pkg/api/v1beta3/conversion.go

```diff
@@ -20,11 +20,2638 @@ import (
 	"fmt"
 
 	newer "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/resource"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/conversion"
 )
 
 func init() {
+	err := newer.Scheme.AddGeneratedConversionFuncs(
```

Comment:
In a followup PR can you add a test that these haven't been manually edited? Thanks :)